src/runtime/components/NuxtTurnstile.vue

```diff
@@ -52,7 +52,6 @@ const unmount = () => {
 }
 
 onMounted(() => {
-  // @ts-expect-error types from nuxt/scripts seem to be wrong
   onLoaded(async ({ render, reset: resetFn, remove: removeFn }) => {
```

Comment:
```suggestion
  // @ts-ignore
  onLoaded(async ({ render, reset: resetFn, remove: removeFn }) => {
```

Reply:
might be that upgrading deps will resolve

Reply:
that's curious, TS does complain locally in turnstile for me. Deps should be identical as there's a lockfile